assets/javascripts/discourse/widgets/discourse-post-event.js

```diff
@@ -178,7 +179,11 @@ export default createWidget("discourse-post-event", {
           endsAt: event.ends_at,
         },
       ],
-      event.recurrence_rule
+      {
+        recurrenceRule: event.recurrence_rule,
+        location: event.url,
+        details: getAbsoluteURL(event.post.url),
```

Comment:
Does this include the hostname? Might need to prepend `document.location.origin`? 🤔